src/Tags/Tagging.hs

```diff
@@ -29,30 +30,41 @@ runTagging blob symbolsToSummarize
   = Eff.run
   . evalState @[ContextToken] []
   . Streaming.toList_
-  . contextualizing blob symbolsToSummarize
+  . contextualizing blob toKind
   . tagging blob
+  where
+    toKind x = do
+      guard (x `elem` symbolsToSummarize)
+      case x of
+        "Function" -> Just Function
+        "Method"   -> Just Method
+        "Class"    -> Just Class
+        "Module"   -> Just Module
+        "Call"     -> Just Call
+        "Send"     -> Just Call
```

Comment:
This is a bug, right? It should be `Just Send`?

Reply:
Nope; we consider `"Send"` to be a kind of `Call`.

Reply:
cf https://github.com/github/semantic/pull/262/files#r329230019

Reply:
Aha! Worth documenting in this code, I think, since it took me by surprise.